Competition2017/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -52,14 +52,15 @@ public void setupMyCommands(
     @Inject
     public void setupClimbingCommands(
             OperatorInterface oi,
-            AscendClimbingCommand ascend,
+            DescendClimbingCommand descend,
+            AscendSafelyCommand ascend,
+            AscendDangerouslyCommand ascendDangerously,
             RopeAlignerCommand aligner)   
     {
         oi.controller.getXboxButton(XboxButton.X).whileHeld(ascend);
         
         oi.controller.getXboxButton(XboxButton.Start).whileHeld(aligner);
-        
-        oi.operatorPanelButtons.getifAvailable(4).whileHeld(ascend);
```

Comment:
Replacing a panel button with an Xbox button? Are we no longer using the panel button?